lib/util/template/index.ts

```diff
@@ -228,7 +228,12 @@ export function compile(
       }
     }
   }
-  return handlebars.compile(template)(filteredInput);
+  try {
+    return handlebars.compile(template)(filteredInput);
+  } catch (err) {
+    logger.warn({ err }, 'Error compiling template');
+    return template;
```

Comment:
Do we want it to return `null`, or the original string?

Reply:
🤔 should we do a test compile on validate instead? so we catch it at config validation?

Reply:
I thought we already had something like that. I think the problem I saw in production is caused by release notes content so is seen at runtime only